src/main/java/org/truffleruby/language/methods/LookupMethodNode.java

```diff
@@ -158,18 +160,18 @@ protected DynamicObject metaClass(Object object) {
     }
 
     protected MethodLookupResult doCachedLookup(VirtualFrame frame, Object self, String name) {
-        return lookupMethodCachedWithVisibility(getContext(), frame, self, name, ignoreVisibility, onlyLookupPublic);
+        return lookupMethodCachedWithVisibility(getContext(), frame, self, name, ignoreVisibility, onlyLookupPublic); // TODO BJF Fix cached lookup
```

Comment:
@eregon I think you worked on the cached lookup and this `TODO` could be removed?

Reply:
Indeed, looks done to me, removed.